app/components/publications/publication/case/contact-persons-panel.js

```diff
@@ -0,0 +1,77 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+
+/**
+ * @argument {PublicationFlow} publicationFlow include: contact-persons,contact-persons.person
+ */
+export default class PublicationsPublicationCaseContactPersonsPanelComponent extends Component {
+  @service store;
+
+  @tracked isOpenAddModal = false;
+
+  get contactPersons() {
+    const contactPersons = this.args.publicationFlow.contactPersons.toArray();
+    // .get() because await is not supported by sort
+    contactPersons.sort((it1, it2) => {
+      const lastName1 = it1.get('person').get('lastName')
+        .toLowerCase();
+      const lastName2 = it2.get('person').get('lastName')
+        .toLowerCase();
+      return lastName1 < lastName2 ? -1 : 1;
+    });
+    return contactPersons;
```

Comment:
Hier kan je `sortBy` gebruiken met dot-notatie voor een nested property. En zo ook eenvoudig als 2e parameter op voornaam sorteren: ` this.args.publicationFlow.contactPersons.sortBy('person.lastName', 'person.firstName')`